### PR TITLE
Contributing

### PR DESCRIPTION
**Pull request by puneith**
Guidance to the OSS community for Kubeflow examples we would like to get started on. This is the copy of the internal doc. This should be the list of examples we would like to see on Kubeflow/examples which will make people successful w/ ML and cover most of the cases / important ones.

Please add Github usernames (Googlers and non-Googlers) who should be aware of this list to start with. 

@jlewi @aronchick @nkashy1 @amygdala.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/examples/6)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Contributing to Kubeflow Examples
+
+You want to contribute to Kubeflow examples? That's awesome! Please refer to the short guide below. 
+
+# Contributing Guide
+
+The [Kubeflow](https://github.com/kubeflow/kubeflow/blob/master/README.md) project is dedicated to making machine learning on Kubernetes simple, portable and scalable. We need your support in making
+this repo the destination for top models and examples, which show the power of Kubeflow. We have created an initial list of
+proposed manifests. Please feel free to self-assign these examples, by following a simple 3 step process:
```
Comment (jlewi) [inline CONTRIBUTING.md]: Why call them manifests and not examples?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* Create a Github issue with the details of the **example** and self-assign
+* Send a PR to this repo with the actual work for the example
+
+We have assigned priorities to the items below. See priority guidance: 
+
+* **P0**: Very important, try to self-assign if there is a P0 available
+* **P1**: Important, try to self-assign if there is no P0 available
+* **P2**: Nice to have
+
+# Proposed Examples
```
Comment (jlewi) [inline CONTRIBUTING.md]: Can you add GitHub Issue summarization?
https://github.com/kubeflow/kubeflow/issues/157

Comment (texasmichelle) [reply CONTRIBUTING.md]: I tried to add a line for #157, but got a permission denied error. @puneith if you add me as a contributor to your fork, I'll push my commit.

Comment (puneith) [reply CONTRIBUTING.md]: Michelle added you as contributor.

File: CONTRIBUTING.md (hunk 1 above)
Comment (puneith) [reply CONTRIBUTING.md]: Okay fixed. Don't want to sound stuck up but want to call out that these are canonical examples / artifacts and not mere some random examples. Models sounds a better word though even that doesn't capture the workflow part of these efforts. Ideas?

File: CONTRIBUTING.md (hunk 2 above)
Comment (texasmichelle) [reply CONTRIBUTING.md]: Added a line for GH issue summarization.

**Comment by texasmichelle** [comment]
What's the deal with this failing check?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+| Example | What does it accomplish? | Priority | Priority reasoning | ML framework | Owner (github_id) | Company | PR link |
```
Comment (ankushagarwal) [inline CONTRIBUTING.md]: We can open issues for each one of these and link them here instead of the the PR link column

**Comment by jlewi** [comment]
I think the tests are failing because we don't have any tests setup for this repository.

If you wanted to add some basic tests you could look at what we do for kubeflow/testing. You could add a test that runs py_lint and py unittests.

Until then you can ignore.

@texasmichelle You have WRITE privileges on this repository. So when you are satisified please merge.

**Comment by texasmichelle** [comment]
OK, I'll work on getting tests setup for this repo, then polish off this PR with issue links and merge.

File: CONTRIBUTING.md (hunk 3 above)
Comment (puneith) [reply CONTRIBUTING.md]: @ankushagarwal sounds good. I have made the fix. @texasmichelle fyi.

**Comment by jlewi** [comment]
Thanks

File: CONTRIBUTING.md (hunk 2 above)
Comment (puneith) [reply CONTRIBUTING.md]: @texasmichelle can you add link to GH issue#

**Pull request merged**